### ISSUE TITLE
for 2023: track registrations locally

### ISSUE THREAD

**Issue by gburgett** [watermarkchurch/clc-app-v2#102]
Rather than dealing with all the caching complexity of querying the server for registration status / "user like" status, track that locally first in the app, and sync it over to the server via background API calls.

1. Use AsyncStorage as primary store for "likes"
2. Doublecheck with server for capacity events before storing the "like" locally
3. In the background, sync over all my liked entries with the server

**Comment by jpowell** [comment]
@gburgett still interested in trying to do this?

**Comment by gburgett** [comment]
Yep!  If we go with Expo then I can totally do this as part of the rewrite.  Should be really simple :)

**Comment by jpowell** [comment]
@gburgett is this done with the new by clc tab?

**Comment by gburgett** [comment]
It is not.  As we discussed, it is more complex than I initially thought.  It would be nice to have but I'm not sure it's worth it.
